phoenix-core/src/main/java/org/apache/phoenix/hbase/index/Indexer.java

```diff
@@ -652,14 +652,18 @@ private void doPostWithExceptions(ObserverContext<RegionCoprocessorEnvironment>
 
   @Override
   public void postOpen(final ObserverContext<RegionCoprocessorEnvironment> c) {
-    Multimap<HTableInterfaceReference, Mutation> updates = failedIndexEdits.getEdits(c.getEnvironment().getRegion());
-    
     if (this.disabled) {
         super.postOpen(c);
         return;
     }
 
+    // if stopped is true, and then init writer
+    if (this.stopped) {
+        initBuilderAndWriter(c.getEnvironment());
```

Comment:
This would mean that we always re-enable a stopped indexer.

I would guess that there are some cases when we don't want to this.

Reply:
only this.stopped is true when region close. when region open again, need to init builder and writer. when region split failed, and then rollback ,do not have init builder and writer.